docs/src/logging.md

```diff
@@ -1,43 +1,50 @@
 # Logging
 
-The operator is designed to log in JSON format directly to standard output,
-including PostgreSQL logs.
+The operator outputs logs in JSON format directly to standard output, including
+PostgreSQL logs, without persisting them to storage for security reasons. This
```

Comment:
I think the more important bit here is this is standard in K8s, and more broadly it's a 12-factor app recommendation.

Reply:
yes, that's exactly what we say in the Troubleshooting page

Reply:
CloudNativePG is fine.